### ISSUE TITLE
Bitcoin Design Community Public Relations

### ISSUE THREAD

**Issue by Bosch-0** [BitcoinDesign/Meta#350]
Creating this issue to keep track of and discuss public relations (PR) efforts of the Bitcoin Design Community (BDC). By PR efforts I'm referring to when community members represent BDC at conferences, on podcasts, meetups etc. 

We should make an effort to ensure community members who take part in these PR efforts have the resources and knowledge to best represent the community. We could do this by creating a slide deck people could use, have stickers, community assets like logos etc.

We should conduct more outreach to conferences, podcasts and other media avenues to talk about BDC. These do not have to be limited to bitcoin-only media but anything that is relevant to us. An example is @ConorOkus @GBKS @johnsBeharry @sbddesign @barefoot-88 and myself recently talking at [COSCUP](https://coscup.org/2022/en/), an open-source conference in Taiwan, about bitcoin and open design. I would personally recommend doing more talks about Bitcoin Design and Bitcoin in general at Web3, cryptocurrency, blockchain events. There is value educating retail  crypto users at the source about bitcoin and "bring them out of the darkness," there is no doubt talented folk in those communities there who could benefit bitcoin, and we could learn from them too.

When there is upcoming events we should find community members to present at them about BDC, preferably ones who are in close proximity so people don't have to travel far. 

### Resources

- [Bitcoin-only conferences](https://bitcoin-only.com/conferences)
- [Bitcoin-only podcasts](https://bitcoin-only.com/podcasts)

**Comment by GBKS** [comment]
Good idea to structure this a bit further. Not sure if I'd call it PR, that sounds very corporate. Is there a better term? Awareness? Spreading the word? Evangelism?

My experiences so far:
- @johnsBeharry and I presented 3 times ([playlist](https://www.youtube.com/playlist?list=PLpV0KfVOMojZXWYX88a4armnUVOQfmzPi)) at open-source conferences (FOSDEM 2021 & 2022, COSCUP 202). Here it's nice that we're already on the same wavelength as the audience as far as the open-source environment and the benefits and disadvantages that come with the arena.
- We also presented at [Figma Config 2021](https://config.figma.com/events/figma-config-2022/home), which lead to a huge influx in Slack. I can't tell how many stuck around to become regular visitors and/contributors.
- @ConorOkus and I participated in a podcast at [Sustain Open Source Design](https://sosdesign.sustainoss.org/19). Great conversation but very hard to tell what effect this had.
- I presented at [Offscript](https://offscript.web3creatives.com) (Web3 creatives event) earlier this year. Various people were already familiar with bitcoin design and were interested in open design and what we do.

As far as resources go:
- [Slide deck](https://www.figma.com/community/file/966708683009348504) from Figma Config 2021 on Figma Community
- [Bitcoin design community logo](https://www.figma.com/community/file/1040606190770628557) on Figma Community
- [Bitcoin design slide template](https://www.figma.com/community/file/972067335916375043) on Figma Community
- [Bitcoin design stickers](https://www.figma.com/community/file/1132217809152964057) on Figma Community

For conferences, I always post publicly in Slack if someone wants to contribute, but get almost no response. I try to track the proposal and talk prep process in Github ([example](https://github.com/BitcoinDesign/Meta/issues/303)). Would love it if more people took initiative there, but I am also happy to do this if others prefer not to.

Now that the pandemic has subsided a bit, it's easier (and safer) to travel and attend in person. I'd like to visit some bitcoin conferences this fall and meet people in person (maybe it's even possible to get speaker slots still?). There's also local UX conference that I am looking at. Let's represent 😎

**Comment by sbddesign** [comment]
> We should make an effort to ensure community members who take part in these PR efforts have the resources and knowledge to best represent the community.

Has this been an issue?

## My experiences

- [PlebFi Austin](https://pleb.fi/austin/) - August 2021 - Spoke at this event with Patricia. Not sure if the video ever materialized.
- [Nashville Bitcoin Meetup](https://www.meetup.com/nashbitcoiners/events/282563388/) - February 2021 - Rod Bitkite and I did sort of a fireside chat type thing about Bitcoin Design. They don't record their meetups. No visuals or presentation, all conversation and audience Q&A.
- [BitDevs NYC](https://bitdevs.org/) - February 2021 - Patricia and I spoke about Bitcoin Design here, and we did a case study on the Blixt wallet design thinking process and a quick Bitcoin UI kit demo. No videos at BitDevs, but we did a [dramatic re-enactment here](https://youtu.be/S8nk-LVMO7w). Here is the [presentation deck](https://www.figma.com/file/MHyOE2RWJHVlfOFMOP8Vfs/Bitcoin-Design---BitDevs-NYC?node-id=0%3A1).
- [PlebFi Miami](https://pleb.fi/miami2022/) - April 2022 - I did a talk for the hackathon participants specifically about using design to improve your hackathon project. Bitcoin Design is referenced heavily throughout the presentation. Not sure if the video has materialized yet. Here is [the slide deck](https://www.figma.com/file/0zgpnr6HnfTD7AJap2VvxD/Bitcoin-Design-PlebFi-Miami-2022?node-id=0%3A1). It's sort of a fork of the NYC deck.
- [Bitcoin 2022](https://b.tc/conference/2022) - April 2022 - @ConorOkus and I participated in the [10xing Bitcoin Development](https://youtu.be/6VMESUZpafo) panel in Miami. It was about BDK, LDK, and Bitcoin Design.
- [PlebFi LA](https://bitdevsla.org/pleb-fi-1/) - June 2022 - I did [a talk about UX for taproot](https://youtu.be/04QmPgR3BUk) at this event. I do mention Bitcoin Design at the beginning, but it's really just my opinions on taproot stuff and not really a full on repping of Bitcoin Design.
- [Asheville Bitcoiners](https://www.meetup.com/asheville-bitcoin-meetup-group/events/287135367/) - August 2022 - I spoke about Bitcoin Design here. This meetup is not recorded. No slide deck. This was more of an informal conversation with the attendees. I kind of just let the conversation flow wherever the audience was more interested in, and it lead more into a discussion about the audience's questions about self-custody and the different options available (and the tradeoffs of those options).
- [Atlanta BitDevs](https://atlantabitdevs.org/) - I've never done a full session dedicated to Bitcoin Design, but I certainly pull up links to sections in the Guide frequently during group conversations.

I've been wanting to do some stuff for [TABConf](https://2022.tabconf.com/), but I haven't really moved on that yet. I'll open an issue for that one.

**Comment by Bosch-0** [comment]
I don't think it has, but if we push for more public outreach I can see it being very valuable to have resources available for those who want to do it.

Ty for sharing all these links!

**Comment by GBKS** [comment]
I am converting this to a discussion as that is more appropriate.
